lasagne/layers/shape.py

```diff
@@ -98,8 +95,8 @@ def get_output_shape_for(self, input_shape, *args, **kwargs):
                                      (o[0], len(input_shape)))
                 output_shape[dim] = input_shape[o[0]]
                 masked_output_shape[dim] = input_shape[o[0]]
-                if ((input_shape[o[0]] is None)
-                    and (masked_input_shape[o[0]] is None)):
+                if (input_shape[o[0]] is None) \
+                   and (masked_input_shape[o[0]] is None):
```

Comment:
Hmm, good point. I usually don't do visual indent there, but indent twice, but I guess pep8 wouldn't like that. I'm fine with line continuation then. The indent level is still pretty close to the line following the if statement now, though... what about a double indent? Is that allowed when we're doing line continuation anyway, and don't have to match visual indent for brackets?